FirebasePerformance/ProtoSupport/README.md

```diff
@@ -0,0 +1,6 @@
+To build the protos:
+- `brew install protobuf`
+- Verify version in generate_protos.sh
+- `./generate_protos.sh`
+
+Make sure to [re-generate the project](https://github.com/firebase/firebase-ios-sdk/tree/perfNpScript/FirebasePerformance#to-develop-on-firebase-performance) after protos are built.
```

Comment:
Update this link from the branch to master

Reply:
Good catch! Done.